src/test/resources/org/opentripplanner/apis/gtfs/queries/feedinfo.graphql

```diff
@@ -0,0 +1,10 @@
+{
+    feeds {
+        agencies {
+	    name
+	    url
+	}
+	publisherUrl
```

Comment:
Not sure if that is overkill but do we perhaps want the following?

```
publisher {
  name
  url
}
```

Reply:
I don't know :)  That looks very graphql like, but does it introduce any advantage? Structuring is not  needed here to automate data access.

Reply:
We'll let the council of elders decide. :)

Reply:
We talked about it in today's dev meeting and @t2gran has requested for the publisher to be a separate object, like I suggested.